### PR TITLE
Shopek to check

### PR DESCRIPTION
**Pull request by AdamProgramista**

### CODE REVIEW

File: src/ShoppingCenter.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ShoppingCenter {
+    String name;
```
Comment (michal-marciniec) [inline src/ShoppingCenter.java:6]: missing `private`

File: src/Shop.java [hunk 2]
```diff
@@ -3,21 +3,35 @@
...
         this.productsList = new LinkedList<>();
-        this.type = type;
+        this.types = types;
         this.servicesList = new LinkedList<>();
         this.employeesList = new LinkedList<>();
     }
+    public Shop(String name, Address address, String[] strings){
+        this(name,address,new LinkedList<String>());
+    }
+    // przeładowanie konstruktora overloading - tworzy shop z pusta listą typów
```
Comment (michal-marciniec) [inline src/Shop.java]: redundant comment

**Review by michal-marciniec — CHANGES_REQUESTED** [review]
sounds good, still better than Yuriy's Pull Request, but some changes are necessary

File: src/ShoppingCenter.java (hunk 1 above)
Comment (AdamProgramista) [reply src/ShoppingCenter.java:6]: you are wrong!!!!!!!!!!!!!!!! :>

Comment (michal-marciniec) [reply src/ShoppingCenter.java:6]: why?

Comment (AdamProgramista) [reply src/ShoppingCenter.java:6]: because. :)

**Review by michal-marciniec — APPROVED** [review]
whoa, man! that's awesome

**Pull request merged**